src/store/custom-content-blocks.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import { store } from 'hybrids';
+
+const CustomContentBlocks = {
+  selectors: store.record([String]),
+  [store.connect]: {
+    get: async () => {
+      const { customContentBlocks = {} } = await chrome.storage.local.get([
+        'customContentBlocks',
+      ]);
+      return { selectors: customContentBlocks };
```

Comment:
would be nice if the key name indicate the we have a map from hostname to selectors. How about `hostnameSelectors` ? or maybe just `hostnames`?

Reply:
As the name of the model will be `ElementPickerSelectors` we can change to `hostnames` here.